packages/widgetbook_cli/bin/parsers/exceptions.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:widgetbook_exception/widgetbook_exception.dart';
+
+class WidgetbookDeployFailure implements WidgetbookException {
```

Comment:
How is the naming convention for MasonCLI? To me, it looks odd that a `WidgetbookDeployFailure` implements a `WidgetbookException`. 

A failure is not an exception.